src/pages/charter.js

```diff
@@ -154,31 +154,30 @@ const CharterPage = ({ location: {pathname} }) => {
         </li>
         <li>The voting members of the BSC consist of:
         <ol>
-          <li>Two representatives (the “Kernel Representatives”) from the group of eBPF Linux
-          kernel maintainers (as specified in the maintainer file). The Kernel Representatives
-          will not be employed by the same Company or by Related Companies.</li>
-          <li>One representative (each a “eBPF Runtime Representative”) for each additional
-          open-source eBPF runtime implementation. Each eBPF Runtime Representative will
-          be nominated and elected by the applicable oversight body of each eBPF runtime,
-          with nominations and elections subject to approval by the BSC. Nominees need to be
-          active contributors to the applicable eBPF runtime.</li>
-          <li>Up to two representatives (“Additional Project Representatives”) from additional
-          open source projects that are important to the eBPF ecosystem, as determined by the
-          BSC. Each Additional Project Representative need to be an active contributor to one
-          of these additional projects. The Additional Project Representatives will not be
-          employed by the same Company or by Related Companies. Each Additional Project
-          Representative must be nominated and elected by the BSC.</li>
-          <li>Up to three representatives with maintainer status (“Maintainer Representatives”) of
-          major eBPF Foundation projects as approved by the BSC. Maintainer Representatives
-          are nominated and elected by the applicable oversight body of each major eBPF
-          Foundation project, with nominations and elections subject to approval by the BSC.
-          No more than one Maintainer Representative may be employed by any one company
-          or group of Related Companies.</li>
+          <li>Two Linux Kernel representatives from the group of eBPF Linux
+          kernel maintainers (as specified in the Linux eBPF maintainer file). The Linux Kernel representatives
+          must not be employed by the same Company or by Related Companies.</li>
+
+          <li>Up to two representatives from other open source eBPF runtime implementations.
+          Each “eBPF Runtime Representative” will represent a different eBPF runtime chosen
+          by the BSC, and must be an active contributor to that eBPF runtime.</li>
+
+          <li>Up to three Technical Project representatives. Each Technical Project
+          Representative candidate must be an active contributor to a different Technical Project
+          chosen by the BSC.</li>
+
+          <li>Up to two eBPF Landscape Project representatives. Each eBPF Landscape Project Representative
```

Comment:
BSC discussion 27 July 2022:
"Landscape Project" isn't well defined.  What about things like LLVM/clang?  Does it have to be on the ebpf.io landscape projects page?   Or can the BSC decide to use a different definition?  Would be helpful to clarify in a future PR.

Reply:
BSC voted in favor of this proposal on July 27th.